ArcGISRuntimeSDKQt_CppSamples/Scenes/ViewContentBeneathTerrainSurface/ViewContentBeneathTerrainSurface.cpp

```diff
@@ -19,17 +19,15 @@
 #include "ArcGISTiledElevationSource.h"
 #include "Scene.h"
 #include "SceneQuickView.h"
-#include "Portal.h"
-#include "PortalItem.h"
+#include "SceneView.h"
```

Comment:
Is this needed? `SceneQuickView` inherits `SceneView`.

Reply:
Good catch! It definitely is not needed.